### PR TITLE
recordkeeping comment

### PR DESCRIPTION
**Pull request by alexis-devwork**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#for heroku deployment
```
Comment (rogusdev) [inline server.py]: whitespace

Comment (alexis-devwork) [reply server.py]: What whitespace?

Comment (rogusdev) [reply server.py]: `# for...` (use backticks reverse apostrophe `)

**Comment by rogusdev** [comment]
Rebase as one commit and force push

**Pull request merged**